acceptance/features/migration.feature

```diff
@@ -1,16 +1,16 @@
-Feature: operator can migrate/adopt Redpanda helm base deployment
+Feature: Helm chart to Redpanda Operator migration
 
   @skip:gke @skip:aks @skip:eks
-  Scenario: Adopt or migrate from stand alone helm chart release to Redpanda custom resource
-        Given a Helm release named "redpanda-migration-example" of local Redpanda Helm Chart with values:
+  Scenario: Migrate from a Helm chart release to a Redpanda custom resource
+        Given a local Helm release named "redpanda-migration-example" with the following values:
```

Comment:
nit: no need for the `following` here. The `:` serves the same purpose

```suggestion
        Given a local Helm release named "redpanda-migration-example" with the values:
```

Reply:
Could this instead be something like `Given a Helm release named "blah" of the "redpanda/redpanda" helm chart`?

That would make it a bit more reusable but also be more clear to non-dev readers. I don't think many people aside from us will understand what "a local Helm release" means.

Reply:
The whole conversation about "local" helm chart started in https://github.com/redpanda-data/redpanda-operator/pull/578#discussion_r2040269823

@chrisseto 
> I think we'll also see a massive amount of benefit if we specify the version of the chart in this test.

@RafalKorepta 
> Currently I'm testing what is locally available. What the acceptance test suite should test only operator, or both operator and Redpanda helm chart?

Should I switch to "latest" released version?

Reply:
I just pushed your suggestion @chrisseto. If pipeline will be green it will auto merge. I can agree it's not flexible/generic implementation, but I will not spend more time on this problem. 

My strong opinion is that we should always test pre-released version that is in the github repo (aka continues integration tests). Which is local Helm chart not the remote `redpanda/redpanda` version.